app/src/main/java/co/develhope/chooseyourownbeer/network/dto/BeersResult.kt

```diff
@@ -1,3 +1,27 @@
 package co.develhope.chooseyourownbeer.network.dto
 
-class BeersResult : ArrayList<BeersDto>()
\ No newline at end of file
+import android.util.Log
+import android.view.View
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import co.develhope.chooseyourownbeer.R
+import co.develhope.chooseyourownbeer.ui.MainActivity
+
+class BeersResult {
+
+
+
+    fun  list(mainActivity: MainActivity, beersResult: BeersResult) {
```

Comment:
perchè in Beerresult c'è la main activity? stai mescolando i layer.

Reply:
dovrei metter quale layer?myApplication, beersProvider o qualcos'altro?